src/index.js

```diff
@@ -0,0 +1,71 @@
+import os from "os";
+import { createInterface } from "readline/promises";
+import { fsPlugin, hashPlugin, osPlugin, zipPlugin } from "./plugins/index.js";
+import { colors } from "./utils/colors.js";
+import { logInvalidOperation } from "./utils/logger.js";
+import { parseCmd, whereIam } from "./utils/util.js";
+
+let username = "Unknown";
+
+const exit = () => {
+  console.log(
+    colors.fgGreen,
+    `${os.EOL}Thank you for using File Manager, ${username}, goodbye!`
+  );
+  process.exit();
+};
+
+const start = () => {
+  const args = process.argv.slice(2);
+
+  args.map((arg) => {
+    if (arg.startsWith("--username=")) {
+      [, username] = arg?.split("=");
+    }
+  });
+  console.log(colors.fgGreen, `Welcome to the File Manager, ${username}!`);
+
+  whereIam();
+
+  // workarround for the Windows CTRL+C hotkey
```

Comment:
Just for your information, this "workaround" just broke your solution.

Reply:
Could you test it under linux/iOs please?

Reply:
I don't have Linux/iOS.

Reply:
> I didn't test it under Windows... Just relied on the StackOverflow. Thank you for the information.

Never rely on StackOverflow answers that were given >8 years ago 😉

Reply:
https://nodejs.org/docs/latest-v18.x/api/process.html#signal-events

Reply:
Please, review it again. I just fixed this Windows `stdin` "workaround".

Reply:
You want -30% from your total score?

Reply:
I don't. But 220 is more than 101, correct?

Reply:
Btw, on the real projects we use WSL to run node. And Docker desktop

Reply:
Okay, as you wish, I'll review it again. The result will be known no later than Friday 😉